includes/admin/views/onboarding-page-one.php

```diff
@@ -19,17 +19,24 @@
 			<div class="simple-podcasting__setting-description"><?php esc_html_e( 'What’s the title of your podcast show that listeners will see?', 'simple-podcasting' ); ?></div>
 		</div>
 
+		<!-- Show artist/author -->
+		<div class="simple-podcasting__setting">
+			<label class="simple-podcasting__setting-label"><?php esc_html_e( 'Show Artist / Author name*', 'simple-podcasting' ); ?></label>
+			<div class="simple-podcasting__setting-input"><input name="podcast-artist" required type="text" /></div>
```

Comment:
```suggestion
			<label class="simple-podcasting__setting-label" for="simple-podcasting-podcast-artist"><?php esc_html_e( 'Show Artist / Author name*', 'simple-podcasting' ); ?></label>
			<div class="simple-podcasting__setting-input"><input name="podcast-artist" id="simple-podcasting-podcast-artist" required type="text" /></div>
```

Accessibility.

You'll also need [aria-describedby ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)for the descriptive text.